### PR TITLE
Update building-linux.md with full instructions

### PR DESCRIPTION
**Pull request by BigTreezZ**
Finally figured out how to compile it via feedback on the Discord server. Making a pull request with all instructions needed so others can compile themselves as well.

### CODE REVIEW

File: documents/building-linux.md [hunk 1]
```diff
@@ -5,6 +5,13 @@ SPDX-License-Identifier: GPL-2.0-or-later
 
 ## Build shadPS4 for Linux
 
+Clone the repository recursively:
+```
+git clone https://github.com/shadps4-emu/shadPS4.git
+cd shadPS4/
+git submodule update --init --recursive
```
Comment (raphaelthegreat) [inline documents/building-linux.md]:
```suggestion
git clone --recursive https://github.com/shadps4-emu/shadPS4.git
cd shadPS4
```

File: documents/building-linux.md [hunk 2]
```diff
@@ -17,5 +24,11 @@ cd build/
 
 Use make to build the project:
 ```
-make -j$(nproc)
+cmake /"PATH"/"TO"/"DIRECTORY"/shadPS4/ && make -j$(nproc)
```
Comment (raphaelthegreat) [inline documents/building-linux.md]:
```suggestion
cmake --build . --parallel
```
One will already be in the build directory if the instructions were followed

**Review by raphaelthegreat — COMMENTED** [review]
Also would be good to add some packages needed to build while you are at it

**Comment by BigTreezZ** [comment]
> Also would be good to add some packages needed to build while you are at it

Sure. I will loosely base the requirements on the RPCS3 build requirements, since it is what I installed on my system, and I had no missing packages needed for compilation.

**Comment by BigTreezZ** [comment]
Done.

**Comment by raphaelthegreat** [comment]
glew library is not needed

**Comment by BigTreezZ** [comment]
Removed it for all distros.

**Review by raphaelthegreat — APPROVED** [review]

**Pull request merged**